controller-client/src/paginator.rs

```diff
@@ -17,6 +17,97 @@ use std::vec::IntoIter;
 use tracing::error;
 use tracing::info;
 
+///
+///Helper method to iterate over the all the Pravega Scopes.
+///This method returns a stream of values, Pravega scopes, produced asynchronously.
+///
+/// The below snippets show case the example uses.
+/// Sample 1:
+///```
+/// # use tonic::transport::Channel;
+/// # use pravega_controller_client::controller::controller_service_client::ControllerServiceClient;
+/// use pravega_controller_client::ControllerClient;
+/// # async fn call_list_scope(controller_client: &dyn ControllerClient) {
```

Comment:
This is typo I suppose?

Reply:
Yes, it is done as part of rustdoc "easy init" mechanism that @tkaitchuck  shared.
I have changed the name even though this will not part of the generated doc and  added the missing `#` for all the methods.